app/models/user.rb

```diff
@@ -18,6 +15,7 @@ def password
   end
 
   def password=(new_password)
+    return @password = new_password if new_password.blank?
```

Comment:
a way to define password as `nil` if none is provided.  Password not required at initial first time registration.